packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -398,6 +399,13 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
         },
       });
 
+      await scheduleWorkflowBodyScan({
```

Comment:
In the update above, I think we need to set `safe` to false

Reply:
But if we do that, then scheduling the updated reminders will fail too (`scheduleWorkflowNotifications` line https://github.com/calcom/cal.com/pull/18822/files#diff-e434eb7fdd98d2062613cf2c8f5a69bd969b932d5cd5b0ca329d434b118f6fa6R413